packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -190,8 +193,44 @@ export default defineComponent({
   name: 'FileDetails',
   setup() {
     const store = useStore()
+    const { $gettext } = useTranslations()
+
+    const copiedDirect = ref(false)
+    const copiedEos = ref(false)
+    const {
+      copy,
+      copied,
+      isSupported: isClipboardCopySupported
```

Comment:
@JammingBen not sure if `isSupported` is needed here/in general in the `v-if`s. I didn't look into it any further but as pointed out in https://github.com/owncloud/web/pull/8136#discussion_r1057366226, from a cursory glance it is always true since we set `legacy: true` in `useClipboard`

Reply:
@dschmidt and I decided to leave it for now, as we are not 100% sure if needed or like (same as you :) )